Source/WebKit/UIProcess/ios/WKPageHostedModelView.mm

```diff
@@ -36,15 +36,49 @@
 #import <pal/spi/cocoa/QuartzCoreSPI.h>
 #import <wtf/RetainPtr.h>
 
+#if __has_include(<CoreRE/CoreRE_SPI_WebKit.h>)
+#include <CoreRE/CoreRE_SPI_WebKit.h>
+#define HAVE_RE_STEREO_CONTENT_SUPPORT RE_FEATURE_EMBEDDED_STEREO_CONTENT_COMPONENT
+#else
+#define HAVE_RE_STEREO_CONTENT_SUPPORT 0
+#endif
```

Comment:
This should be defined in `wtf/PlatformHave.h` and then used in this file as  `HAVE(RE_STEREO_CONTENT_SUPPORT)`.